.pylintrc

```diff
@@ -9,6 +9,7 @@ init-hook='import sys; sys.path.append("../oppia_tools/google_appengine_1.9.67/g
 
 # Checks for correct docstring style
 load-plugins=pylint.extensions.docstyle, pylint_quotes, scripts.pylint_extensions
+ignore-patterns=.*_test.py
```

Comment:
If this has no effect, please remove it.

Reply:
It's ignoring all the test files ending with `_test.py`. So, we can't remove it. Thanks!